diff --git a/node_modules/react-native-raw-bottom-sheet/index.d.ts b/node_modules/react-native-raw-bottom-sheet/index.d.ts
index 31a5259..1c4cf13 100644
--- a/node_modules/react-native-raw-bottom-sheet/index.d.ts
+++ b/node_modules/react-native-raw-bottom-sheet/index.d.ts
@@ -1,4 +1,4 @@
-import { Component } from "react";
+import { Component, PropsWithChildren } from "react";
 import { StyleProp, ViewStyle } from "react-native";
 
 declare module "react-native-raw-bottom-sheet" {
@@ -20,7 +20,7 @@ declare module "react-native-raw-bottom-sheet" {
       draggableIcon?: StyleProp<ViewStyle>;
     };
     keyboardAvoidingViewEnabled?: boolean;
-  };
+  } & PropsWithChildren;
 
   export default class RBSheet extends Component<RBSheetProps> {
     open(): void;
